src/main/resources/hudson/tasks/Mailer/global.jelly

```diff
@@ -24,38 +24,40 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <f:section title="${%E-mail Notification}">
-    <f:entry title="${%SMTP server}" field="smtpHost">
-      <f:textbox />
-    </f:entry>
-    <f:entry title="${%Default user e-mail suffix}" field="defaultSuffix">
-      <f:textbox />
-    </f:entry>
-    <f:advanced>
-
-      <f:optionalProperty title="${%Use SMTP Authentication}" field="authentication"/>
-
-      <f:entry title="${%Use SSL}" field="useSsl">
-        <f:checkbox />
-      </f:entry>
-      <f:entry title="${%Use TLS}" field="useTls">
-        <f:checkbox />
-      </f:entry>
-      <f:entry title="${%SMTP Port}" field="smtpPort">
+  <l:hasPermission permission="${app.MANAGE}" >
```

Comment:
Is this needed? I think the permission check is already done at this point (the descriptor is not going to be shown anyways because there is an early permission check when loading descriptors to show).

Reply:
probably not, will take a look and remove